zipkin-server/README.md

```diff
@@ -268,6 +268,13 @@ The following apply when `STORAGE_TYPE` is set to `elasticsearch`:
                            to 0 as it would mean a machine failure results in data loss.
     * `ES_USERNAME` and `ES_PASSWORD`: Elasticsearch basic authentication, which defaults to empty string.
                                        Use when X-Pack security (formerly Shield) is in place.
+    * `ES_SECURITY_FILE_PATH`: Elasticsearch security properties file path, which defaults to empty
```

Comment:
Wondering if instead of the more generic `SECURITY` term, `ES_CREDENTIALS_FILE` would be a more descriptive and concise wording given that it only contains credentials ? Unless the file is going to be used for other dynamically reloadable security properties, will that be the case ?

Reply:
Make sense. The file doesn't intend to load any other items except for username/password. I will update the term then fix all of the references.

Reply:
done